src/constants/hypixel/Hypixel.js

```diff
@@ -260,4 +260,221 @@ export const HYPIXEL = {
 		'stats.SkyClash.wins', // SkyClash (removed)
 		'stats.WoolGames.wool_wars.stats.wins',
 	],
+	MODES: {
+		ARCADE: {
+			"DAYONE": 'Blocking Dead',
```

Comment:
```suggestion
			DAYONE: 'Blocking Dead',
```
Please remove quotation marks from the keys in this file.

Reply:
Got it. Looking back, I think I kept keys without quotations, however for a key like 1v1 or 2v2, it errors.
![image](https://github.com/user-attachments/assets/cb7746d1-017a-483d-99f6-227eb2014536)
What should I do about keys that start with numbers?

Reply:
Ahh, in that case, you can single quote them → `'1v1'`